checks/utils/file_utils.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package checks
+package utils
```

Comment:
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

FYI...Dave is well respected in the Golang community. Here is his advice which valued lots of people. I am referring to using the `utils` package. Here is an example of Dave's work https://github.com/pkg/errors which let go 1.13 error handling changes https://go.dev/blog/go1.13-errors

Also from Rob Pike https://go-proverbs.github.io `A little copying is better than a little dependency.`

What are your thoughts?

Reply:
Gotcha. Will move it under `checks/worker` since that's where it's going to be used. Thanks for the suggestion!

Reply:
Thank you! 😊